### PR TITLE
Sharing is Caring: Masks Galore

### PR DESCRIPTION
**Pull request by KasparoVy**
:cl:
rscadd: Vox versions of the Officer SWAT sechailer + HOS SWAT sechailer + Warden SWAT sechailer.
rscadd: Resprited versions of most Human masks to fit Unathi, Tajara and Vulpkanin.
rscadd: Warden now gets their own version of the SWAT sechailer in their locker.
tweak: HOS now gets the appropriate version of the SWAT sechailer in their locker.
tweak: Repositioned smokeables now available for Vox, Unathi, Tajara and Vulpkanin.
tweak: Properly shades Vox bandana down-state sprites.
/:cl:

**Adds Vox versions of the Officer SWAT sechailer + HOS SWAT sechailer + Warden SWAT sechailer.**

**Tweaks the Vox face-covering bandana sprites, adding improved shading.**

**Adds versions of the following, re-sprited to fit Unathi, Tajara and Vulpkanin:**
- Balaclava, up and down state
- All bandana colours (+ skull pattern), up and down states
- Breath mask, up and down state
- Clown mask+wig
- Cyborg mask
- Death Commando mask
- Gas mask
- All wrestling masks
- Medical mask, up and down state
- Mime mask
- Owl mask
- Plague doctor mask
- Ninja mask
- Sechailer, up and down state
- Sexy Clown mask+wig
- Sexy Mime mask+wig
- Sterile/Surgical mask, up and down state
- SWAT mask
- Welding gas mask, visor up and down state
- Officer SWAT sechailer + HOS SWAT sechailer + Warden SWAT sechailer
- All smokeables re-positioned so non-humans aren't smoking through their cheek.

**Also fixes some typos, does a tiny bit of cleaning and gives the Warden their sechailer.**
The sprite was just laying around, it seems.

All sprites tested and referenced against appropriate head templates, final adjustments made based on in-game results.

_It'll take a while to get pictures up,_ Over 38 masks * 2 directions being featured * 3 species + (3 vox masks * 2 directions being featured) = At least 234 dummies I need to equip and pose. _It is very time consuming_

**One of many pictures, more to come.**
_**Note, some sprites are not shown as they are reskins of others and to save time. i.e. coloured bandanas.**_

_Vox SWAT Sechailers and all new Welding Helmets._
voxops weldinghelmets

_Unathi masks, frontal view._
unathimasksfront

_Unathi masks, side view._
unathimasksright

_Vulpkanin masks, frontal view._
vulpmasksfront

_Vulpkanin masks, side view._
vulpmaskside

_Tajaran masks, frontal view._
tajaramasksfront

_Tajaran masks, side view._
tajaramasksside

### CODE REVIEW

File: code/modules/clothing/masks/gasmask.dm [hunk 1]
```diff
@@ -207,10 +197,23 @@
...
 	ignore_maskadjust = 0
-	species_fit = list()
-	species_fit = list("Vox")
 	action_button_name = "HALT!"
 
+/obj/item/clothing/mask/gas/sechailer/hos
+	name = "\improper HOS SWAT mask"
+	desc = "A close-fitting tactical mask with an especially aggressive Compli-o-nator 3000. It has a tan stripe."
+	icon_state = "hosmask"
+	aggressiveness = 3
```
Comment (Fox-McCloud) [inline code/modules/clothing/masks/gasmask.dm]: Further proof his actual title is the Head of Shitcurity ;p

**Comment by Fox-McCloud** [comment]
Changelog

**Comment by NTSAM** [comment]
cyborg mask more like catborg mask
:+1:

**Comment by KasparoVy** [comment]
Changelog ahoy

**Comment by KasparoVy** [comment]
I'll make the preview pictures early Monday, I've got deadlines to meet
Sorry! Sprites can always be viewed in the .dmi files and pasted on over the template heads I've included if you want a better idea of how they'll look.

**Comment by KasparoVy** [comment]
Alright, we're good now. Thanks for putting the label on!

I was looking back through the sprites at work and noticed an area that could be improved on the balaclavas, and that I'd forgotten to add smokables. Both sorted

**Comment by Ar3nn** [comment]
Vox can't smoke mostly because they have to wear a breath mask at all times to have their nitrogen.

**Comment by KasparoVy** [comment]
I know, I figured I'd just pose that question for good measure. I only ever play Vox, but I'm not the only Vox player

**Comment by VampyrBytes** [comment]
@KasparoVy @Ar3nn 
If the question is 'is it possible for Vox to smoke', the answer is yes. It's probably pretty rare, but if a Vox gets NO_BREATHE then they can get rid of the mask and smoke

**Comment by Crazylemon64** [comment]
That, or they get a bored atmos tech to fill some room with nitrogen for them...

Don't ask me how they actually LIGHT the damn thing without oxygen, though...

**Comment by Ar3nn** [comment]
bluespace

**Comment by KasparoVy** [comment]
That's it, all the pictures are up and I've run out of things to tweak.

**Pull request merged**